src/main/java/nl/thehyve/ocdu/factories/PatientDataFactory.java

```diff
@@ -27,6 +27,8 @@
     public final static String STUDY = "Study";
     public final static String SITE = "Site (optional)";
     public final static String[] MANDATORY_HEADERS = {STUDY_SUBJECT_ID, STUDY, SITE};
+    public static final String[] ALL_PERMITTED_COLUMNS = {STUDY_SUBJECT_ID, STUDY, SITE, GENDER,
+            DATE_OF_BIRTH, DATE_OF_ENROLLMENT, PERSON_ID, SECONDARY_ID};
```

Comment:
no. It is about what is permitted - so if a column name not present in this array is found - this will give error.